src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/MatchXpathCheckTest.java

```diff
@@ -147,7 +147,7 @@ public void testCheckWithDoubleBraceInitialization()
                 + "                )])]");
         checkConfig.addMessage("matchxpath.match", "Do not use double-brace initialization");
```

Comment:
How to take care of these configs? Shall we add support for them in our parser?

Reply:
https://github.com/checkstyle/checkstyle/issues/10545